domains/authorization/Authorization.API/API/Data/Repository.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using API.Models;
+
+namespace API.Data;
+
+public class Repository<T>(ApplicationDbContext context) : IRepository<T>
```

Comment:
All methods are not Async. So thread will be blocked.

Reply:
Fixed.